simple_network_sim/loaders.py

```diff
@@ -86,12 +109,29 @@ def readNodeAttributesJSON(filename):
 # in future - e.g. sampling, generating movements, whatever
 # it should return a networkx graph, ideally with weighted edges
 # eventual replacement with HDF5 reading code?
-def genGraphFromContactFile(filename):
-    G = nx.read_edgelist(filename, create_using=nx.DiGraph, delimiter=",", data=(('weight', float),))
+def genGraphFromContactFile(filename: str) -> nx.DiGraph:
+    """Read a file containing edge weights between nodes.
+
+    Pairs of nodes are listed in the file by source, destination and weight.
```

Comment:
sorce, destination, weight and adjustment.

Reply:
Fixed with c51b8957f0a6bc1649b643e9ee296c75172b4296